src/shared/ui/Switch/index.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '~@styles/colors.scss';
+
+.root {
+  position: relative;
+  display: inline-block;
+  width: 44px;
+  height: 24px;
+  border-radius: 8px;
+  overflow: hidden;
+  cursor: pointer;
+
+  input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+  }
+
+  .switch {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(243, 243, 243, 1);
+    -webkit-transition: .4s;
```

Comment:
maybe we can use autoprefixer? does next.js provide one by default?

Reply:
nextjs has this on the box, just copied from figma